.changeset/shy-buses-mix.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-react': minor
+---
+
+refactor `createAsyncValidators` to be recursive
```

Comment:
Whilst this is true, its a side effect of fixing a bug 👍 Can we make this a little more specific as to what we've fixed now and that we support more complex schema definitions or something? :pray: